circus/commands/addwatcher.py

```diff
@@ -76,7 +76,7 @@ def execute(self, arbiter, props):
         # escalation
         if arbiter.endpoint_owner_mode:
             cmd_uid = options.get('uid', None)
-            if cmd_uid != arbiter.endpoint_owner:
+            if cmd_uid not in arbiter.endpoint_owner_ids:
```

Comment:
I think we should have two options, one for the uid and one for the username. Mixing both is confusing.